HowTo.md

```diff
@@ -216,31 +216,29 @@ Select operation and press Enter. Ctrl+C for cancel
 
 ```
 
-Select an operation to make the node produce a QR-code for it:
+Select an operation to make the node produce a JSON file for it:
 ```
-QR code was saved to: /tmp/dc4bc_qr_2f217f58-a94f-47d8-b871-f35a15275184.gif
+JSON file was saved to: /tmp/3086f09822d7ba4bfb9af14c12d2c8ef_2f217f58-a94f-47d8-b871-f35a15275184.json
 ```
 
-Open the GIF-animation in any gif viewer and take a video of it:
+Open the JSON-animation in any gif viewer and take a video of it:
 ```
-open -a Safari /tmp/dc4bc_qr_2f217f58-a94f-47d8-b871-f35a15275184-request.gif
+open -a Safari /tmp/3086f09822d7ba4bfb9af14c12d2c8ef_2f217f58-a94f-47d8-b871-f35a15275184.json
 ```
 
-After that, you need to scan the GIF. To do that, you need to open the `./qr_reader_bundle.html` in your Web browser on an airgapped machine (firefox from plaintext media in case of Tails airapped machine setup), allow the page to use your camera and demonstrate the recorded video to the camera. After the GIF is scanned, you'll see the operation JSON. Click on that JSON, and it will be saved to your Downloads folder.
+After that, you need to transform the JSON to QR GIF. To do that, you need to open the `./qr_reader_bundle.html` in your Web browser on an airgapped machine (firefox from plaintext media in case of Tails airapped machine setup), allow the page to use your camera and demonstrate the recorded video to the camera. After you should transform the JSON to QR codes. You need to pull your JSON file to the encoder on an airgapped machine and decode it from QR to JSON on the hot node machine.
```

Comment:
this part a little bit misleading.
As far as i understand, new workflow is
Sender(hot or airgapped):
1) Open json file in the app https://github.com/lidofinance/qr-tool (it will be compiled to one html)
2) Generate GIF file
3) Show it to opposite machine

Receiver(the opposite machine)
1) Open app https://github.com/lidofinance/qr-tool in a browser
2) Read GIF and save it to JSON